cranelift/2025/cranelift-11-05.md

```diff
@@ -44,9 +44,11 @@
     the host if other stores/instances also have access to a shared memory
 
 - Cranelift as a production rustc backend
-  - main issue is performance with unwinding enabled
+  - one main issue is performance with unwinding enabled
   - (discussion about performance implications of more blocks)
-  - bjorn3: 7% compile-perf impact with unwinding
+  - jorn3: 7% compile-perf impact with unwinding (but: this includes enabling
+    MIR and Cranelift optimizations; unclear what just unwinding does)
```

Comment:
Still not quite right. The 7% is optimizations enabled, but without unwinding.

Reply:
Ah, sorry -- happy to rubber-stamp a PR to update the notes if you want to get it phrased correctly!

Reply:
https://github.com/bytecodealliance/meetings/pull/704